src/Umbraco.Infrastructure/DependencyInjection/UmbracoBuilder.CoreServices.cs

```diff
@@ -200,7 +200,6 @@ public static IUmbracoBuilder AddCoreInitialServices(this IUmbracoBuilder builde
             // Add default ImageSharp configuration and service implementations
             builder.Services.AddSingleton(SixLabors.ImageSharp.Configuration.Default);
             builder.Services.AddSingleton<IImageDimensionExtractor, ImageSharpDimensionExtractor>();
```

Comment:
Normally we add a NoopImpementation so you can boot Infrastructure without the web assemblies